test/wasm2js/sign_ext.wast

```diff
@@ -5,4 +5,10 @@
   (func "test16" (param $x i32) (result i32)
     (i32.extend16_s (local.get $x))
   )
+  (func "test8_i64" (param $x i64) (result i64)
+    (i64.extend8_s (local.get $x))
+  )
+  (func "test16_i64" (param $x i64) (result i64)
+    (i64.extend16_s (local.get $x))
+  )
```

Comment:
I think we also want a test for `ExtendS32Int64`, the 32-bit case?

Reply:
Done.